controller/ItemResultPreviewer.php

```diff
@@ -149,10 +149,8 @@ protected function getItemResultVariables(core_kernel_classes_Resource $delivery
     {
         $variable = $this->getItemVariable($delivery, $resultIdentifier, $itemDefinition);
         if (isset($variable['uri'])) {
-            $responses = ResponseVariableFormatter::formatStructuredVariablesToItemState([$variable]);
-            if (isset($responses[$variable['uri']])) {
-                return $responses[$variable['uri']];
-            }
+            $responses = $this->getResultVariableStructureHandler()->format([$variable]);
```

Comment:
are we sure that it will work?

Reply:
It doesn't work

Reply:
yes we should have only `return current($responses);`